src/Core/AdminConsole/Models/Data/EventIntegrations/HecIntegration.cs

```diff
@@ -0,0 +1,5 @@
+﻿#nullable enable
+
+namespace Bit.Core.AdminConsole.Models.Data.EventIntegrations;
+
+public record HecIntegration(string Scheme, string Token, Uri Uri);
```

Comment:
⛏️: Reorder the parameters to be consistent with `WebhookIntegration(Uri Uri, string? Scheme = null, string? Token = null);`